docs/design/specs/Ecma-335-Augments.md

```diff
@@ -1,4 +1,4 @@
-# Known Issues in ECMA-335 CLI Specification 
+# Additions to the ECMA-335 CLI Specification
 
 This is an informal list of notes on issues that have been encountered
 with the ECMA-335 CLI specification, primarily during development,
```

Comment:
@jkotas Given that Aleksey is looking for a vetted runtime spec for module initializers, would you suggest any changes to this title and the introductory paragraphs yet?

Reply:
What about:

This is a list of additions and edits to be made in ECMA-335 specifications. It includes both documentation of new runtime features and issues encountered during development.